far2l/vtshell.cpp

```diff
@@ -1019,7 +1019,7 @@ static bool shown_tip_exit = false;
 			return;
 	}
 	
-	~VTShell()
+	virtual ~VTShell()
```

Comment:
but why? its not really needed here - there're no descedants of VTShell, and  if make its d-tor virtual then better to same fot c-tors of interfaces that it derived from